code/modules/hydroponics/plant_genes.dm

```diff
@@ -251,21 +251,28 @@
 	// Also affects plant batteries see capatative cell production datum
 	name = "Electrical Activity"
 	rate = 0.2
+	COOLDOWN_STATIC_LIST_DECLARE(recent_victims)
 
 /datum/plant_gene/trait/cell_charge/on_slip(obj/item/reagent_containers/food/snacks/grown/G, mob/living/carbon/C)
+	if(!COOLDOWN_LIST_FINISHED(src, recent_victims, C)) // if you were a victim recently, you won't get it again
+		return
 	var/power = round(G.seed.potency*rate)
 	if(prob(power))
+		COOLDOWN_LIST_START(src, recent_victims, C, 2 SECONDS)
 		C.electrocute_act(power, G, 1, 1)
 		var/turf/T = get_turf(C)
 		if(C.ckey != G.fingerprintslast)
 			C.investigate_log("[C] has slipped on an electric plant at [AREACOORD(T)]. Last fingerprint: [G.fingerprintslast].", INVESTIGATE_BOTANY)
 			log_combat(C, G, "slipped on and got electrocuted by", null, "with the power of 10. Last fingerprint: [G.fingerprintslast]")
 
 /datum/plant_gene/trait/cell_charge/on_squash(obj/item/reagent_containers/food/snacks/grown/G, atom/target)
+	if(!COOLDOWN_LIST_FINISHED(src, recent_victims, target)) // if you were a victim recently, you won't get it again
+		return
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
 		var/power = G.seed.potency*rate
 		if(prob(power))
+			COOLDOWN_LIST_START(src, recent_victims, C, 2 SECONDS)
```

Comment:
These don't make a lot of sense to have cooldowns on, they execute only on slip and squash which are discrete and non-repeatable events

Reply:
It can happen when multiple items are thrown to a person, or when you step on a squashable slip plant. That's why CD check exists here